<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Garfield</class>
 <widget class="QWidget" name="Garfield">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>359</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Garfield</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWebView" name="webView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>910</width>
     <height>280</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="url">
    <url>
     <string>https://garfield.com/uploads/strips/1978-06-19.jpg</string>
    </url>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>891</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Date</string>
      </property>
      <property name="buddy">
       <cstring>dateEdit</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="minimumDate">
       <date>
        <year>1778</year>
        <month>6</month>
        <day>17</day>
       </date>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="calcButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="todayButton">
      <property name="text">
       <string>Today</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="showButton">
      <property name="text">
       <string>Show</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>891</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Date axis</string>
      </property>
      <property name="buddy">
       <cstring>horizontalSlider</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="horizontalSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Realtime</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dateEdit</tabstop>
  <tabstop>todayButton</tabstop>
  <tabstop>showButton</tabstop>
  <tabstop>horizontalSlider</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>webView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>webView</sender>
   <signal>loadProgress(int)</signal>
   <receiver>progressBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
